release/RELEASE.md

```diff
@@ -113,34 +113,51 @@ This file lists all the steps to follow when releasing a new version of ODS Andr
 
 - Click `Integrations`, find the uploaded bundle in the `Artifacts` sections and click `Create release`.
 
-    ![Create OMA Portal release](images/google_play_store_release_01.png) 
+  ![Create OMA Portal release](images/google_play_store_release_01.png)
 
 - Progress through the various submission steps.
 
 - When step is `Store submissions`, select `Closed beta with Google Groups`, set `Email addresses` to `odsapp@googlegroups.com` and click `Go for store submission`.
 
-    ![OMA Portal store submission](images/google_play_store_release_02.png)<br /><br />
+  ![OMA Portal store submission](images/google_play_store_release_02.png)<br /><br />
+
+- To publish the release you have to click a last time on "Got to publication". You should receive a confirmation email from OMA with a "Published" status.
+
+### Announce the new release on GitHub & FoODS
+
+- Go to [GitHub ods-android - Discussions > Announcements](https://github.com/Orange-OpenSource/ods-android/discussions/categories/announcements)
```

Comment:
```suggestion
- Go to [GitHub Announcements](https://github.com/Orange-OpenSource/ods-android/discussions/categories/announcements)
```

Because we wrote "Go to GitHub XXX" for other links ("Go to GitHub Actions" and "Go to GitHub Releases" for instance)